deployment/terraform/create.go

```diff
@@ -399,27 +399,3 @@ func (t *Terraform) getOutput() (*terraformOutput, error) {
 	}
 	return &output, nil
 }
-
-func (t *Terraform) displayInfo(output *terraformOutput) {
-	mlog.Info("Deployment complete. Here is the setup information:")
-	mlog.Info("Proxy server: " + output.Proxy.Value.PublicDNS)
-	mlog.Info("Instances:")
-	for _, instance := range output.Instances.Value {
-		mlog.Info(instance.PublicIP)
-	}
-	mlog.Info("Agents:")
-	for _, agent := range output.Agents.Value {
-		mlog.Info(agent.Tags.Name + ": " + agent.PublicIP)
-	}
-	if len(output.Agents.Value) > 0 {
```

Comment:
So, this was for: 
```
Mattermost URL: http://
Instances:
Load Agents:
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/mattermost/mattermost-load-test-ng/deployment/terraform.(*Terraform).displayInfo(0xc000199d00, 0xc00040e000)
	/Users/ibrahim/go/src/github.com/mattermost/mattermost-load-test-ng/deployment/terraform/info.go:36 +0xb45
...
```

Reply:
Yes. I removed it because we already validate this in the config that agents have to be > 0. Do you still see a need to do this ? Unless somehow a bad deployment has happened and there are no agents. Maybe it's necessary after all.

Reply:
Yeah, an edge case maybe but there may be an information that we might be missing in the end.

Reply:
done